frontend/src/index.tsx

```diff
@@ -23,9 +23,11 @@ if (typeof window !== 'undefined') {
     }
 }
 
-ReactDOM.render(
-    <Provider store={getContext().store}>
-        <App />
-    </Provider>,
-    document.getElementById('root')
-)
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Would appreciate a comment (explanation or just a link to the Slack thread) a ton